docs/design/logs.md

```diff
@@ -0,0 +1,240 @@
+# OpenTelemetry Rust Logs Design
+
+Status:
+[Development](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md)
+
+## Overview
+
+OpenTelemetry (OTel) Logs support differs from Metrics and Traces as it does not
+introduce a new logging API for end users. Instead, OTel recommends leveraging
+existing logging libraries such as `log` and `tracing`, while providing bridges
```

Comment:
```suggestion
existing logging libraries such as [log](https://crates.io/crates/log) or `[tracing](https://crates.io/crates/tracing)`, while providing bridges
```

Reply:
It might also be worth pointing out that tracing is not "distributed tracing", but rather in-process-structured-logging-and-execution-traces.

Because the name makes it easy to think it is something it is not